sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -569,3 +569,27 @@ public abstract partial class ObjectSerializer
         public virtual System.Threading.Tasks.ValueTask<System.BinaryData> SerializeAsync(object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
 }
+namespace Azure.Messaging
+{
+    public partial class CloudEvent
+    {
+        public CloudEvent(string source, string type, System.BinaryData data, string dataContentType, Azure.Messaging.CloudEventDataFormat dataFormat = Azure.Messaging.CloudEventDataFormat.Binary) { }
+        public CloudEvent(string source, string type, object jsonSerializableData, System.Type? dataSerializationType = null) { }
+        public System.BinaryData? Data { get { throw null; } set { } }
+        public string? DataContentType { get { throw null; } set { } }
+        public string? DataSchema { get { throw null; } set { } }
+        public System.Collections.Generic.IDictionary<string, object?> ExtensionAttributes { get { throw null; } }
+        public string Id { get { throw null; } set { } }
+        public string Source { get { throw null; } set { } }
+        public string? Subject { get { throw null; } set { } }
+        public System.DateTimeOffset? Time { get { throw null; } set { } }
+        public string Type { get { throw null; } set { } }
+        public static Azure.Messaging.CloudEvent? Parse(System.BinaryData jsonEvent, bool strict = true) { throw null; }
+        public static Azure.Messaging.CloudEvent[] Parse(string jsonContent, bool strict = true) { throw null; }
+    }
+    public enum CloudEventDataFormat
+    {
+        Json = 0,
```

Comment:
Should we reorder these items so `default(CloudEventDataFormat)` would be Binary?

Reply:
Yeah I suppose that makes more sense.